reporter/src/main/kotlin/reporters/SpdxDocumentReporter.kt

```diff
@@ -48,11 +48,6 @@ class SpdxDocumentReporter : Reporter {
         private const val DOCUMENT_NAME_DEFAULT_VALUE = "Unnamed document"
     }
 
-    enum class FileFormat(val reportFilename: String) {
```

Comment:
I believe this breaks the references from the kDoc of the `SpdxDocumentReporter` class. It could be fixed by just importing `SpdxModelMapper.FileFormat`. Importing would also shorten below code from e.g. from ` enumValues<SpdxModelMapper.FileFormat>` to `enumValues<FileFormat>`.

One short-coming of this change is that the public API of the reporter now contains this enum class from ´spdx-utils´. Do you believe the change is still worth it? (e.g. the API is nicer but the amount of callers is very low).

Reply:
> I believe this breaks the references from the kDoc of the `SpdxDocumentReporter` class.

Good point, fixed.

> One short-coming of this change is that the public API of the reporter now contains this enum class from ´spdx-utils´. Do you believe the change is still worth it? (e.g. the API is nicer but the amount of callers is very low).

Hmm, I'm not sure I'm able to follow. The reporter still uses the `enum` only internally, in the implementation of `generateReport`, but not in its signature. So IMO the `spdx-utils` module still only is an implementation dependency of the `reporter` module, not an API dependency.

BTW, this implementation dependency on `spdx-utils` was not explicit so far, and I've added another commit to fix that.

Reply:
> Hmm, I'm not sure I'm able to follow. The reporter still uses the enum only internally, ...

Ah right, good point.